### PR TITLE
docs(call activity): add `propagateAllParentVariables`

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

closes: #2979 

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by korthout — APPROVED** [review]
Thanks @nicpuppa 👍 

👍 The documentation looks good from a technical viewpoint, but I see a few options to clarify the text. Please have a look before merging.

**Review by christinaausley — DISMISSED** [review]

**Comment by nicpuppa** [comment]
Hey @korthout I applied your suggestions, please if you have time have another look 👀 🙇

File: docs/components/modeler/bpmn/call-activities/call-activities.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ If the attribute `propagateAllChildVariables` is set (default: `true`), all vari
 
 It's recommended to disable the attribute `propagateAllChildVariables` or define output mappings if the call activity is in a parallel flow (e.g. when it is marked as [parallel multi-instance](../multi-instance/multi-instance.md#variable-mappings)). Otherwise, variables can be accidentally overridden when they are changed in the parallel flow.
 
+By default, all variables of the call activity scope are copied to the created process instance. This can be limited to copying only the local variables of the call activity, by setting the attribute `propagateAllParentVariables` to `false`.
+
+By disabling this attribute, variables existing at higher scopes are no longer copied. If the attribute `propagateAllParentVariables` is set (default: `true`), all variables are propagated to the child process instance.
+
```
Comment (korthout) [inline docs/components/modeler/bpmn/call-activities/call-activities.md:46]: 🔧 You can move this up and replace line 35 entirely

**Review by korthout — APPROVED** [review]
LGTM 👍 

🔧 One last suggestion

**Comment by akeller** [comment]
Enabling auto-merge so we can include this with a release ASAP - https://camunda.slack.com/archives/C026U8GBNSW/p1704811372613899

**Review by christinaausley — APPROVED** [review]

**Pull request merged**